vng_api_common/descriptors.py

```diff
@@ -83,7 +83,8 @@ def __set__(self, obj, value: Optional[dict]):
 
         # map the values
         for key, field in self.mapping.items():
-            setattr(obj, field.name, value[key])
+            if key in value:
+                setattr(obj, field.name, value[key])
```

Comment:
Looking at the code above, I think if the value is missing, we should rather set the default value instead of doing nothing.